.travis.yml

```diff
@@ -13,7 +13,24 @@ install:
   - pip install sphinx
 script:
   - tox
-  
+
 branches:
   only:
   - master
+
+jobs:
+  include:
+    - stage: deploy
+      python: 3.6
+      env:
+        - TOXENV=py36-sphinx18
+      deploy:
+        # production pypi
+        - provider: pypi
+          distributions: sdist bdist_wheel
+          user: deply_username
+          password:
+            secure:
+          on:
+            branch: master
```

Comment:
What is the purpose of `branch: master` here? We don't want to release to PyPI on master merges, only on new tags.

Reply:
Its just an example - we need to fill in password and username as well for this to work.

Reply:
I can help with the pypi credentials before merge.

I had to look this one up. `branch: master` here defines _which branch to run the command on_. It does not specify that pushes to master trigger a new release.